ddprof-ffi/cbindgen.toml

```diff
@@ -12,6 +12,11 @@ style = "both"
 no_includes = true
 sys_includes = ["stdbool.h", "stddef.h", "stdint.h"]
 
+after_includes = """
+
+#define ddprof_ffi_CharSlice_from_literal(string) \\
```

Comment:
Any objection to `DDPROF_FFI_CHARSLICE_C` instead? It's shorter, in upper-case like macros should be, and uses the `_C` convention which is used in standard macros for integer literals eg `INT64_C(1000000000000)`.

Reply:
I went ahead and pushed a commit. If you object, feel free to revert. I was just trying to speed up the speculative happy path.